resources/scripts/export-kibana-dashboards.sh

```diff
@@ -0,0 +1,20 @@
+export_dot_env() {
```

Comment:
provide a few lines explaining how to use this script, 
and and add a Makefile targets